index.js

```diff
@@ -159,21 +159,24 @@ function launchSpecFromConnectionInfo(
   connectionFile,
   spawnOptions
 ) {
-  const argv = kernelSpec.argv.map(
-    x => (x.replace("{connection_file}", connectionFile))
+  const argv = kernelSpec.argv.map(x =>
+    x.replace("{connection_file}", connectionFile)
   );
 
   const defaultSpawnOptions = {
     stdio: "ignore",
     cleanupConnectionFile: true
   };
-  const env = Object.assign({}, process.env, kernelSpec.env);
-  const fullSpawnOptions = Object.assign(
```

Comment:
Can we move this logic to a `createSpawnOptions` function that we can use to do testing?

Reply:
I'm going to assume since this outer part is tested now that this comment is out of date.